tests/TestCli.cpp

```diff
@@ -944,6 +957,56 @@ void TestCli::testGenerate()
     QCOMPARE(m_stderrFile->readLine(), QByteArray("Invalid password length bleuh\n"));
 }
 
+void TestCli::testImport()
+{
+    Import importCmd;
+    QVERIFY(!importCmd.name.isEmpty());
+    QVERIFY(importCmd.getDescriptionLine().contains(importCmd.name));
+
+    QScopedPointer<QTemporaryDir> testDir(new QTemporaryDir());
+    QString databaseFilename = testDir->path() + "testImport1.kdbx";
+
+    Utils::Test::setNextPassword("a");
+    importCmd.execute({"import", m_xmlFile->fileName(), databaseFilename});
+
+    m_stderrFile->reset();
+    m_stdoutFile->reset();
+
+    QCOMPARE(m_stdoutFile->readLine(),
+             QByteArray("Insert password to encrypt database (Press enter to leave blank): \n"));
+    QCOMPARE(m_stdoutFile->readLine(), QByteArray("Successfully imported database.\n"));
+
+    Utils::Test::setNextPassword("a");
+    auto db = QSharedPointer<Database>(Utils::unlockDatabase(databaseFilename, true, "", "", Utils::DEVNULL));
+    QVERIFY(db);
```

Comment:
@jsachs is there an entry in the xml database? we could add a basic assert here on the presence of the entry.